src/devtools/client/debugger/src/reducers/source-actors.ts

```diff
@@ -164,9 +164,25 @@ function updateBreakableLines(
     return state;
   }
 
-  return updateResources(state, [{ id: sourceId, breakableLines: value }]);
+  return updateResources(state, [{ breakableLines: value, id: sourceId }]);
 }
 
+const mergeRanges = (ranges: { min: number; max: number }[]): { min: number; max: number }[] => {
```

Comment:
:hand: Might be worth pulling out the `{min, max}` type so it doesn't have to get repeated inline in each spot

Reply:
Yeah I was also thinking about this, let's do it 👍